src/utils/inputValidation.js

```diff
@@ -0,0 +1,10 @@
+export const inputHasValue = (value) => {
+	return value.trim().length === 0 ? false : true;
+};
+
+export const hasSameValue = (string, value) => {
+	const regex = /[!"#$%&'()*+,-./:;<=>?@[\]^_`{|}~\s]/g;
```

Comment:
Hey!

When I console logged this regex, I noticed some things 'leaked' - like numbers, the € and £ symbols and am wondering if it wouldn't be better to do this instead:

/[^a-z]/g

as this would match anything that is not a-z and when calling replace(regex,'') the letters would remain and wouldn't have random numbers in it if the user makes an input mistake.

Reply:
Hello!! Thanks for the suggestion. I'm working on changing it!